config/initializers/session_store.rb

```diff
@@ -1,3 +1,7 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_catchup-web_session'
+Rails.application.config.session_store(
+  :cookie_store,
+  key: '_catchup-web_session',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.